tasks/install-apt.yml

```
@@ -6,33 +6,70 @@
     name:
       - ca-certificates
       - apt-transport-https
+      - gpg
```

With `deb822_repository` we only need this package as dependency

```suggestion
      - python3-debian
```